x/ibc/07-tendermint/types/client_state.go

```diff
@@ -144,6 +144,15 @@ func (cs ClientState) VerifyClientConsensusState(
 		return err
 	}
 
+	if consensusState == nil {
```

Comment:
isn't this checked on `sanitizeVerificationArgs`?

Reply:
I took it out of `sanitizeVerificationArgs` since the consensusState is now being retrieved and decoded from the store